apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/PreAdjClaimResponseTypeTransformerV2.java

```diff
@@ -0,0 +1,20 @@
+package gov.cms.bfd.server.war.commons;
+
+import com.codahale.metrics.MetricRegistry;
+import org.hl7.fhir.r4.model.ClaimResponse;
+
+/**
+ * A {@link FunctionalInterface} describing the types that handle transforming Fiss/MCS records into
```

Comment:
small thing, but FISS (Financial Intermediary Standard System) and MCS (Mulit-Carrier System) are both acronyms